jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,811 @@
+---
+title: "Version 1.6.0 released"
+author: The Nim Team
+---
+
+We are very proud to announce Nim version 1.6, the latest stable release, after a year in the making!
+This is by far the biggest release yet, here are some stats:
+* 1667 PRs merged (1760 commits)
+* 893 issues closed
+* 15 new stdlib modules
+* new features in >40 stdlib modules, including major improvements to 10 commonly used modules.
+* documentation and minor improvements to 170 modules, including 312 new runnable examples
+* 280 new nimble packages
```

Comment:
```suggestion
Nim version 1.6 is now officially released!

A year in the making, 1.6 is the latest stable release and by far the largest yet. We're proud of what we, the core team and dedicated volunteers, have accomplished with this milestone. If you're new to Nim, it's a statically typed compiled systems programming language; and version 1.6 makes it all that much more compelling, visit the [Nim homepage](https://nim-lang.org/) to learn more.

Again, none of this would have been possible without the core team, volunteers, and community who's combined efforts resulted in:
* 1667 PRs merged (1760 commits)
* 893 issues closed
* 15 new stdlib modules
* new features in >40 stdlib modules, including major improvements to 10 commonly used modules.
* documentation and minor improvements to 170 modules, including 312 new runnable examples
* 280 new nimble packages
```

If you'd like a bit more flair or "marketing" in the writing.

Reply:
i took some of your suggestions, but not all:
> If you're new to Nim, it's a statically typed compiled systems programming language; and version 1.6 makes it all that much more compelling, visit the [Nim homepage](https://nim-lang.org/) to learn more.

the problem here is this repeats what's mentioned a few lines below ("building on Nim's core strenghs:...")

> Again, none of this would have been possible without the core team, volunteers, and community who's combined efforts resulted in:

this kind of repeats what's said a few lines earlier ("we, the core team and dedicated volunteers"); see also "Contributors to Nim 1.6" section appearing later

we can iterate ...